EventFilter/CSCRawToDigi/python/cscPacker_cfi.py

```diff
@@ -10,12 +10,17 @@
 # packer - simply get rid of it
 from Configuration.Eras.Modifier_run2_common_cff import run2_common
 run2_common.toModify( cscpacker,
-                      useFormatVersion = 2013,
+                      formatVersion = 2013,
                       usePreTriggers = False,
                       packEverything = True)
```

Comment:
if I understand correctly, CSC packing is only enabled for Run3 with this PR. Have you studied also enabling it for Run2?

Reply:
Indeed it is, we have not yet considered re-enabling it for Run-2.

Reply:
The packing is of course always enabled (DIGI->RAW)  :) But yes, the readout suppression  (dropping digis based on preLCT requirements and a range of BX) could be enabled for run2, but I don't think it's useful to do so now - haven't all simulations for run2 already been run?

In fact the default WAS for readout suppression to be enabled, but it was turned off with a run2 era modification. Please talk to Slava K. who decided on that, and indeed simulation life is simpler, if not most realistic, without the suppression. 

We really need it ON for the future though, because of course it's becoming important that for phase2 simulated RAW event size model real RAW event size as closely as possible so that we can better understand event size under phase2 conditions.

Reply:
Yes, I should have clarified that it's the zero-suppression. Thanks Tim.

Reply:
Thanks for the clarification. My question was indeed prompted by a discussion with @slava77, who suggested to ask whether zero-suppression should now be enabled for Run2 (for possible future rereco) as well as Run3, in order to maintain only one path.

Reply:
I suppose that's up to CSC DPG to decide.

Reply:
cc @cms-sw/csc-dpg-l2

Reply:
I AM a CSC DPG L2 and I already answered: We (CSC) are more than happy to have readout suppression simulated for run 2, since that is more realistic.

Reply:
ok, let me put this in an issue and go forward with it now.